### ISSUE TITLE
Acess the loss of all trials not only the one with best parameters

### ISSUE THREAD

**Issue by JanoschMenke** [facebook/Ax#285]
Hi,

another question. How would I be able to get the loss of all trials of an experiment set up for an `AxClient`.  
Lets say:
```
ax= AxClient(enforce_sequential_optimization=False)
ax.create_experiment(
    name="hyperop",
    parameters=parameters,
    objective_name="loss",
    minimize=True
    )
```
I would like to get a list or dictionary of all the trials.

**Comment by JanoschMenke** [comment]
Literally found the answer a second later:

For people search it is:
`ax.get_trials_data_frame()`

**Comment by ldworkin** [comment]
Yep, that's exactly right @JanoschMenke ! Out of curiosity, were you following a particular tutorial or documentation? We do have this documented here: https://ax.dev/tutorials/gpei_hartmann_service.html#4.-Run-optimization-loop but maybe it's missing somewhere else?

**Comment by JanoschMenke** [comment]
I was not working through the Tutorials. I think the problem is, that you cant really be sure which Tutorial contains what information.  
For example if you are just looking for a simple Hyperparameteroptimization then I would probably first follow the https://ax.dev/tutorials/tune_cnn.html Tutorial which does not include that information. I would not know what to expect from the Service API Tutorial, but maybe that is just the way that I approached it.

I think a Documentation which does not only rely on Tutorials, could be helpful, but I understand that it is a lot more work.

I just tried to tab through all possible function that belonged to `ax` and found it .
